eZ/Publish/API/Repository/PermissionService.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+use eZ\Publish\API\Repository\Values\User\UserReference;
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * This service provides methods for permission related operations.
+ */
+interface PermissionService
+{
+    /**
+     * Get current user.
+     *
+     * Loads the full user object if not already loaded, if you only need to know user id use {@see getCurrentUserReference()}
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    public function getCurrentUser();
+
+    /**
+     * Get current user reference.
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\UserReference
+     */
+    public function getCurrentUserReference();
+
+    /**
+     * Sets the current user to the given $user.
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\UserReference $user
+     */
+    public function setCurrentUser(UserReference $user);
```

Comment:
Renamed to `setCurrentUserReference` in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/8a67e929c7792499b51b5a33c957233fbbdad124.